templates/deployment_jobs.yaml

```diff
@@ -163,6 +163,15 @@ spec:
         {{- end }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
+        {{- if and (regexMatch "^[0-9.]+$" .Values.image.tag) (semverCompare ">=2.110.0" .Values.image.tag) }}
+        livenessProbe:
+          httpGet:
+            path: /api/checkJobsRunnerHealth
+            port: 3003
```

Comment:
Is this the right port to use? Asking because I don't see `3003` mentioned anywhere else in this repo.

Reply:
Yeah it's the right one, it's referenced in the main retool_development repo. We're choosing a new port to avoid any conflicts that might arise in an all-in-one setup (even though we aren't adding liveness probes to that setup, just to be safe in case a customer hacks it themselves).

Reply:
Awesome, thanks for the details.